src/plugins/dashboard/public/application/legacy_app.ts

```diff
@@ -56,6 +74,7 @@ export function initDashboardApp(app, deps) {
       ['createItem', { watchDepth: 'reference' }],
       ['getViewUrl', { watchDepth: 'reference' }],
       ['editItem', { watchDepth: 'reference' }],
+      ['editItemAvailable', { watchDepth: 'reference' }],
```

Comment:
What is this prop for?

Reply:
The prop `editItemAvailable` will be passed through into the `<TableListView` component, where the supplied row-Fn is used to determine if the "actions" icons should be displayed.  
This is added to allow registered list-providers self-determine if their target OSD App instances have an URL-addressable "Edit" mode.
Those providers that do not have a url-addressable "edit", such as Observability Apps, can provide a static "false" via this fn, and prohibit the display of the "edit" action icon.